app/javascript/src/apps/admin/UserManagement.js

```diff
@@ -1461,22 +1492,26 @@ export default class UserManagement extends React.Component {
           {renderConfirmButton(g.type !== 'Device' && (g.confirmed_at == null || g.confirmed_at.length <= 0), g.id)}
           {renderReConfirmButton(g.unconfirmed_email, g.id)}
         </td>
-        <td className="col-md-2 py-3">
+        <td className="py-3">
           {g.name}
         </td>
-        <td className="col-md-1 py-3">
+        <td className="py-3">
           {g.initials}
         </td>
-        <td className="col-md-2 py-3">
+        <td className="py-3">
           {g.email}
         </td>
-        <td className="col-md-1 py-3">
+        <td className="py-3">
           {g.type}
         </td>
-        <td className="col-md-2 py-3">
+        <td className="py-3">
+          {g.available_space === 0 ? ''
+            : `${Math.round((g.used_space / g.available_space) * 100)}% of ${g.available_space / 1024 / 1024}MB`}
```

Comment:
Add space before "MB", otherwise the unit is difficult to see.
E.g., trying the feature in the UI I got `0% of 0.09999942779541016MB`.